registry/mainnet/interchain/squid.tokenlist.json

```
@@ -10,31 +10,41 @@
     {
       "tokenId": "0xb689de707e78469c6a115fdcfd9a0c2db9f8e7ce5ade7b29526d63f4e01d77fb",
       "tokenAddress": "0x524d524B4c9366be706D3A90dcf70076ca037aE3",
+      "tokenManagerAddress": "0x5F53895484af488e27fd33CE70a3dbD745aeAA52",
+      "tokenManagerType": "mint_burn",
```

```suggestion
      "tokenManagerType": "mintBurn",
```
since we're using camel case everywhere?